actions/kpops-runner/README.md

```diff
@@ -11,7 +11,7 @@ This action runs KPOps with the given config.
 | kpops-version     |    ✅    |       -       | string | KPOps version to use                                                                                                                          |
 | working-directory |    ❌    |       .       | string | root directory used by KPOps to run pipelines                                                                                                 |
 | pipeline-base-dir |    ❌    |       .       | string | directory where relative pipeline variables are initialized from                                                                              |
-| defaults          |    ❌    |   defaults    | string | defaults folder path                                                                                                                          |
+| defaults          |    ❌    |       .       | string | defaults folder path                                                                                                                          |
```

Comment:
Unrelated to this PR?

Reply:
true, I will move it into a separate one

I did it to make manual testing simpler